client/src/elm/Pages/Tuberculosis/ProgressReport/Model.elm

```diff
@@ -0,0 +1,32 @@
+module Pages.Tuberculosis.ProgressReport.Model exposing (..)
+
+import Backend.Entities exposing (..)
+import Components.ReportToWhatsAppDialog.Model
+import EverySet exposing (EverySet)
+import Pages.Page exposing (Page)
+
+
+type alias Model =
+    { showEndEncounterDialog : Bool
+    , reportToWhatsAppDialog : Components.ReportToWhatsAppDialog.Model.Model
+
+    -- , components : Maybe (EverySet Components.ReportToWhatsAppDialog.Model.ReportComponentTuberculosis)
```

Comment:
Will it be done in this PR or in follow-ups?

Reply:
Waiting for @adamhstewart to update on https://github.com/TIP-Global-Health/eheza-app/issues/1207#issuecomment-2194351795